packages/web/pages/guilds.tsx

```diff
@@ -18,6 +19,19 @@ export const getStaticProps = async () => {
 
 const GuildsPage: React.FC<Props> = ({ guilds }) => (
   <PageContainer>
+    <Head>
+      <meta name="title" content="Metagame's Guilds" />
+      <meta
+        name="description"
+        content="A Massive Online Coordination Game! Guilds participating in Metagame."
+      />
+      <meta property="og:title" content="Metagame's Guilds" />
+      <meta
+        property="og:description"
+        content="A Massive Online Coordination Game! Guilds participating in Metagame."
+      />
+      <meta property="og:url" content="https://my.metagame.wtf/guilds" />
+    </Head>
```

Comment:
Would be nice to turn this into a component with props for title / description etc that have defaults. Will ensure consistency between pages that they all use the same tags

Reply:
I'll look into making components :+1:

Do we need to pull in some plugin like [next-seo](https://github.com/garmeeh/next-seo)?

Reply:
> I'll look into making components 👍
> 
> Do we need to pull in some plugin like [next-seo](https://github.com/garmeeh/next-seo)?

Not necessary but that might make it easier to implement, feel free to use it

Reply:
I created a component, in [`components/Seo.tsx`](https://github.com/Vyvy-vi/TheGame/blob/feat/og-meta-tags/packages/web/components/Seo.tsx) - commit - 7ebb213

Is the implementation alright?